include/seqan3/alignment/scoring/gap_scheme.hpp

```diff
@@ -26,8 +26,8 @@ namespace seqan3
 // seqan3::gap_score
 // ------------------------------------------------------------------
 
-/*!\brief A strong type of underlying type `score_type` that represents the score of any character against a gap
- *        character. 
+/*!\brief [DEPRECATED] A strong type of underlying type `score_type` that
+ *        represents the score of any character against a gap character.
  * \deprecated This type is deprecated and will be removed in SeqAn3.1.
  * \tparam score_type The underlying type.
  * \ingroup scoring
```

Comment:
You marked this as deprecated, but `struct SEQAN3_DEPRECATED_310 gap_score` isn't deprecated, is there a reason for that?

Reply:
Ah, nice, I missed that. :-).  git log shows that gap_score was marked deprecated already last year see commit 54a2ea6a